### DISCUSSION TITLE
Why aren't there any installers?

### DISCUSSION THREAD

**Discussion by beedell-roke** [orbitalquark/textadept#395] [Q&A] [answered]
I'd love to be able to install and use Textadept, but since I can't register any file associations reliably with it, it's a nuisance to use usually since it's just not convenient. Navigating somewhere to launch it and then using a file picker to locate the file is just worse.

Could we at least have the undermentioned:

| Type    | OS                |
| ------- | ----------------- |
| `.Deb`  | Debian, and forks |
| `.RPM`  | Fedora and *SUSE  |
| `.MSIX` | Windows 7 to 12   |
| `.DMG`  | macOS             |

…to cover the primary platforms?

**Comment by orbitalquark** [comment]
What OS are you on? Normally you can right-click > Open With..., find 
Textadept, and hopefully your OS will make the connection so you don't 
have to do this every time.

**Reply by beedell-roke** [reply]
@orbitalquark, I tend to reorganize frequently enough that this is problematic. I don't know what goes into an installer, but Linux installers are usually just a `tar` file inherently with a little metadata file.

**Comment by georgeraraujo** [comment]
If you're on Windows, you can adapt the instructions in the [Notepad++ User Manual](https://npp-user-manual.org/docs/shell-extension/#alternatives) and have a convenient "Edit With Textadept" context menu for all files.

**Reply by beedell-roke** [reply]
@georgeraraujo, I'm not, and I don't particularly want to do bother with manual stuff like that, since I need to rather frequently change which software is installed on the systems I administrate and own. Thanks, though.

**Reply by ggiesen** [reply]
I'm with @RokeJulianLockhart, the lack of package files is a huge annoyance.

**Comment by rgieseke** [comment, marked as answer]
For many projects installers or packages are maintained by other people using these platforms.

@paaguti has a something for Debian packages, for example: .https://github.com/paaguti/debian_ta

**Reply by ggiesen** [reply]
FYI I've made both DEB and RPM packages available at https://ggiesen.gitlab.io/textadept-pkg/ 

Would love to hear some feedback from folks.
